src/legacy/core_plugins/embeddable_api/public/panel/_embeddable_panel.scss

```diff
@@ -0,0 +1,198 @@
+/**
+  * EDITING MODE
+  * Use .dshLayout--editing to target editing state because
+  * .embPanel--editing doesn't get updating without a hard refresh
+  */
+
+$dshEditingModeHoverColor: transparentize($euiColorWarning, lightOrDarkTheme(.9, .7));
+
+.embPanel {
+  z-index: auto;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  position: relative;
+
+  // SASSTODO: The inheritence factor stemming from embeddables makes this class hard to change
+  .panel-content {
+    display: flex;
+    flex: 1 1 100%;
+    min-height: 0; // Absolute must for Firefox to scroll contents
+  }
+
+  .embeddable-root {
```

Comment:
good catch, i think this can just go away.  I did add `overflow: auto` to it though.  Otherwise my new error embeddables which use the error callout spill over.